examples/chart/teleport-cluster/tests/__snapshot__/auth_deployment_test.yaml.snap

```diff
@@ -141,7 +141,7 @@ should set nodeSelector when set in values:
     - args:
       - --diag-addr=0.0.0.0:3000
       - --apply-on-startup=/etc/teleport/apply-on-startup.yaml
-      image: public.ecr.aws/gravitational/teleport-distroless:16.0.0-dev
+      image: public.ecr.aws/gravitational/teleport-distroless:16.0.0-dev.noah.tbr.1
```

Comment:
this is replacing the version with a wrong tag

Reply:
Right yeah - I'm going to rebase out the [2a812ee](https://github.com/gravitational/teleport/pull/38259/commits/2a812ee67d50fe2d7aeb0e96de3d1605ed012739) commit once everything is approved and I don't need to cut any more releases :)